Templates/Service Management/ServiceNow/HTTP to ServiceNow- Indicent/long-description.md

```diff
@@ -0,0 +1,53 @@
+This template process receives an HTTP POST request containing incident information, and creates an incident into ServiceNow based on it.
```

Comment:
Up for preference, but could something like "...and creates a corresponding incident in ServiceNow." be nicer?